File: src/main/java/guru/xgm/language/jass/psi/tree/JassIElement.java
package guru.xgm.language.jass.psi.tree;

import guru.xgm.language.jass.lang.JassLanguage;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class JassIElement extends IElementType {

    public JassIElement(@NotNull @NonNls String debugName) {
        super(debugName, JassLanguage.Companion.getInstance());
    }

    @Override
    public String toString() {
        return "JassIElement." + super.toString();
    }
}

File: src/main/kotlin/guru/xgm/language/jass/psi/tree/JassIElement.kt
package guru.xgm.language.jass.psi.tree

import com.intellij.psi.tree.IElementType
import guru.xgm.language.jass.lang.JassLanguage.Companion.instance
import org.jetbrains.annotations.NonNls

class JassIElement(debugName: @NonNls String) : IElementType(debugName, instance) {
    override fun toString(): String {
        return "JassIElement." + super.toString()
    }
}
